ember-container-query/src/components/container-query.hbs

```diff
@@ -1,4 +1,3 @@
-{{! @glint-ignore: ember-element-helper needs to provide Glint signature }}
```

Comment:
[Dan on Discord](https://discord.com/channels/480462759797063690/1126182801876713532/1126191306109091840):

> it looks like a bug
>
> specifically, if I set `noEmitOnError: false`, declarations are emitted
>
> I'm guessing it happens to work with `--build` because that will always emit declarations, regardless of errors (because `--build` prioritizes having declarations available for your other project references)
>
> ah
>
> it's because of the `@glint-ignore` here https://github.com/ijlee2/ember-container-query/blob/remove-rollup-plugin-ts/ember-container-query/src/components/container-query.hbs#L1
> 
> the bug appears to be that TS internally is counting that as an error that triggers `noEmitOnError` behavior
>
> if I add an entry for `element` in the registry in `unpublished-development-types`, everything works